day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,26 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping
+    toppings << "Sour Cream"
+  end
+
+  def remove_topping
+    toppings.pop(2)
+  end
+
+  def change_protein
+     protein << "Fish and beans"
```

Comment:
What does running `puts dinner.change_protein` result in when `protein == "Beans"`?